src/lib.rs

```diff
@@ -190,7 +192,6 @@ pub trait LogLevel {
 }
 
 /// Default to [`log::Level::Error`]
-#[allow(clippy::exhaustive_structs)]
```

Comment:
Why is this needed?

Reply:
It's not, which is why it's removed.

Reply:
That is independent of adding log levels and should be its own commit

Reply:
e628191